kiali/README.md

```diff
@@ -86,4 +105,5 @@ When you're done, you can remove the example and all its resources from the clus
 
     ```bash
     helm delete -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME}
+    kubectl -n ${KYMA_KIALI_NS} delete -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
```

Comment:
Pardon me, I forgot that it works with or without the curly brackets when I added the comment. It must have been because of the other error I got that I just connected the wrong dots. 
But either way, without the brackets is the convention we use in Kyma documentation for environment variables, and we use the brackets for placeholders, so it's still good that you removed them. I just wanted to correct myself :)